### DISCUSSION TITLE
record_transformer adding quotes to my json payload.

### DISCUSSION THREAD

**Discussion by ccovarrubias-paradigm** [fluent/fluentd#4161] [Q&A]
Hello all. I was hoping to get some help figuring out what's probably a super easy fix. I am running my logs through the `record_transformer`  filter to inject some specific keys. My `record["message"]` content is json.
```
<filter mylogs.raw>
  @type record_transformer
  enable_ruby true
  renew_record true
  <record>
    message \{${"\"log_type\": \"application_name\", \"log_level\": \"" + record["loglevel"] + "\", \"payload\":" + record["message"]}\}
  </record>
</filter>
```
The problem I'm running into is after I run it through the filter, fluentd is wrapping the value in quotes (` { message: "{key...`) where as I want the value to be native json. Is there a way in the filter to remove the double quotes?

**Comment by kenhys** [comment]
It seems that `message ${{"log_type": "application_name", "log_level": record["loglevel"], "payload": record["message"]}}` is enough.

Here is the sample to emulate:

```
<source>
  @type dummy
  tag mylogs.raw
  dummy {"loglevel": "info", "message": {"key": "value"}}
</source>

<filter mylogs.raw>
  @type record_transformer
  enable_ruby true
  renew_record true
  <record>
    message ${{"log_type": "application_name", "log_level": record["loglevel"], "payload": record["message"]}}
  </record>
</filter>

<match **>
 @type stdout
</match>
```

outcome is:

```
2023-05-02 17:55:40.034551079 +0900 mylogs.raw: {"message":{"log_type":"application_name","log_level":"info","payload":{"key":"value"}}}
```
